kitsune/questions/config.py

```diff
@@ -140,37 +140,37 @@
                         # TODO: Just use the IA topics for this.
                         # See bug 979397
                         (
-                            "download-and-install",
+                            "install-and-update",
                             {
-                                "name": _lazy("Download, install and migration"),
-                                "topic": "download-and-install",
-                                "tags": ["download-and-install"],
+                                "name": _lazy("Install and Update"),
+                                "topic": "install-and-update",
+                                "tags": ["install-and-update"],
                             },
                         ),
                         (
-                            "privacy-and-security",
+                            "protect-your-privacy",
                             {
                                 "name": _lazy("Privacy and security settings"),
-                                "topic": "privacy-and-security",
-                                "tags": ["privacy-and-security"],
+                                "topic": "protect-your-privacy",
+                                "tags": ["protect-your-privacy"],
                             },
                         ),
                         (
                             "customize",
```

Comment:
Same thought as above here, does this need to be `customize-settings-and-preferences` instead?